build_projects/shared-build-targets-utils/Utils/HostVersion.cs

```diff
@@ -63,10 +63,10 @@ public override string ToString()
         // Full versions and package information.
         public string LatestHostBuildMajor => CommitCountString;
         public string LatestHostBuildMinor => "00";
-        public bool EnsureStableVersion => false;
-        public VerInfo LatestHostVersion => new VerInfo(1, 2, 0, ReleaseSuffix, LatestHostBuildMajor, LatestHostBuildMinor, CommitCountString);
-        public VerInfo LatestHostFxrVersion => new VerInfo(1, 2, 0, ReleaseSuffix, LatestHostBuildMajor, LatestHostBuildMinor, CommitCountString);
-        public VerInfo LatestHostPolicyVersion => new VerInfo(1, 2, 0, ReleaseSuffix, LatestHostBuildMajor, LatestHostBuildMinor, CommitCountString);
+        public bool EnsureStableVersion { get; set; }
+        public VerInfo LatestHostVersion => new VerInfo(Major, Minor, Patch, ReleaseSuffix, LatestHostBuildMajor, LatestHostBuildMinor, CommitCountString);
```

Comment:
@schellap We will never make breaking changes in patch version updates. Also, completely breaking changes will only be done in major version updates (in sync with SemVer semantics). Also, when we are dealing with non-Zero patch version, we are pretty much in servicing for that major.minor version combination.